recipes-bsp/u-boot/u-boot-boundary-common_2018.07.inc

```diff
@@ -4,7 +4,7 @@ LIC_FILES_CHKSUM = "file://Licenses/README;md5=30503fd321432fc713238f582193b78e"
 
 PV = "v2018.07+git${SRCPV}"
 
-SRCREV = "ca185fc7406207da613c728935f534a4054674f1"
+SRCREV = "f45f2571cecb47799fd3a3d8b428f7e56a2fa824"
```

Comment:
Add the commit logs on the commit message or explain what bump is this.

Reply:
Added logs on commit message